DiamondLightSource/mx_bluesky
# rename I24 to i24

fixes #143 

Also needs https://github.com/DiamondLightSource/dodal/pull/731

for consistency with beamline modules elsewhere (e.g. in dodal) and python conventions generally

to test:
- check references to "I24" in python modules or corresponding directories have been updated
- run tests
